.env

```diff
@@ -0,0 +1,11 @@
+# Environment variables declared in this file are automatically made available to Prisma.
+# See the documentation for more detail: https://pris.ly/d/prisma-schema#accessing-environment-variables-from-the-schema
+
+# Prisma supports the native connection string format for PostgreSQL, MySQL, SQLite, SQL Server, MongoDB and CockroachDB.
+# See the documentation for all the connection string options: https://pris.ly/d/connection-strings
+
+DATABASE_URL="postgresql://testdb_owner:XhzqGw9ZJ0jB@ep-purple-water-a5qm2how-pooler.us-east-2.aws.neon.tech/challengauthchallenge?sslmode=require"
```

Comment:
this file should not have been committed in the repository at all, it should be kept a secret

Reply:
i suggest you find a way to get rid of it, there's a few guides online on how to remove and purge a committed file